### PR TITLE
docs: output of COPY INTO TABLE.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

after https://github.com/datafuselabs/databend/pull/12594

- Closes #issue

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/12605)

### CODE REVIEW

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md [hunk 1]
```diff
@@ -208,6 +208,19 @@ copyOptions ::=
 When importing large volumes of data, such as logs, it is recommended to set both `PURGE` and `FORCE` to True. This ensures efficient data import without the need for interaction with the Meta server (updating the copied-files set). However, it is important to be aware that this may lead to duplicate data imports.
 :::
 
+### Output
+
+The command returns the following columns:
```
Comment (soyeric128) [inline docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:213]:
```suggestion
COPY INTO provides a summary of the data loading results with these columns:
```

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md [hunk 2]
```diff
@@ -208,6 +208,19 @@ copyOptions ::=
...
+
+| Column           | DataType | Nullable | Description                                |
+|------------------|----------|----------|--------------------------------------------|
+| FILE             | VARCHAR  | No       | The relative path to the source file       |
```
Comment (soyeric128) [inline docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:217]:
```suggestion
| FILE             | VARCHAR  | NO       | The relative path to the source file.       |
```

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md [hunk 3]
```diff
@@ -208,6 +208,19 @@ copyOptions ::=
...
+| ROWS_LOADED      | INT      | NO       | Number of rows loaded from the source file |
```
Comment (soyeric128) [inline docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:218]:
```suggestion
| ROWS_LOADED      | INT      | NO       | The number of rows loaded from the source file. |
```

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md [hunk 4]
```diff
@@ -208,6 +208,19 @@ copyOptions ::=
...
+| ERRORS_SEEN      | INT      | NO       | Number of error rows in the source file    |
```
Comment (soyeric128) [inline docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:219]:
```suggestion
| ERRORS_SEEN      | INT      | NO       | The number of error rows found in the source file.    |
```

Comment (soyeric128) [reply docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:219]: By the way, can we rename ERRORS_SEEN to ERRORS_FOUND?

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md [hunk 5]
```diff
@@ -208,6 +208,19 @@ copyOptions ::=
...
+| FIRST_ERROR      | VARCHAR  | YES      | First error of the source file             |
```
Comment (soyeric128) [inline docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:220]:
```suggestion
| FIRST_ERROR      | VARCHAR  | YES      | The first error found in the source file.             |
```

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md [hunk 6]
```diff
@@ -208,6 +208,19 @@ copyOptions ::=
...
+| FIRST_ERROR_LINE | INT      | YES      | Line number of the first error             |
```
Comment (soyeric128) [inline docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md:221]:
```suggestion
| FIRST_ERROR_LINE | INT      | YES      | Line number of the first error.             |
```

File: docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md (hunk 1 above)
Comment (youngsofun) [reply docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md]: COPY INTO （table）and COPY INTO （stage ）have diff schemas

**Review by soyeric128 — APPROVED** [review]

**Pull request merged**